e2etest/l2tests/assignment.go

```diff
@@ -200,13 +204,24 @@ var _ = ginkgo.Describe("IP Assignment", func() {
 					AllocateTo: &metallbv1beta1.ServiceAllocation{Priority: 20, Namespaces: []string{f.Namespace.Name}},
 				},
 			}
+			testNs, err := cs.CoreV1().Namespaces().Get(context.Background(), f.Namespace.Name, metav1.GetOptions{})
+			framework.ExpectNoError(err)
+			if testNs.Labels == nil {
```

Comment:
given that the framework always append some labels, I'd remove this and assume the map is set and empty, and I would just do the assignement here without the function